handler.cpp

```diff
@@ -273,4 +273,156 @@ RequestHandler::Status StatusHandler::HandleRequest(const Request& request, Resp
     return RequestHandler::PASS;
 }
 
+//////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////// Proxy Handler /////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////////////
+
+RequestHandler::Status ProxyHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+
+    for (unsigned int i = 0; i < config.statements_.size(); i++){
+        // get the root from the child block
+        if (config.statements_[i]->tokens_[0] == "host" && config.statements_[i]->tokens_.size() == 2){
+            m_host = config.statements_[i]->tokens_[1];
+        }
+        else if (config.statements_[i]->tokens_[0] == "port" && config.statements_[i]->tokens_.size() == 2){
+            std::cout << "port: " << m_port << std::endl; 
+            m_port = config.statements_[i]->tokens_[1];
+        }
+    }
+    std::cout << "host: " << m_host << std::endl;
+    std::cout << "port: " << m_port << std::endl;
```

Comment:
These can be taken out now that it works correct? If its too verbose then the production server logs/stdout will be polluted with debugging statements